### PR TITLE
Send wallet

### PR DESCRIPTION
**Pull request by matiu**
Transfer fund between wallets using a popup in `send`

### CODE REVIEW

**Comment by matiu** [comment]
updated and ready for review!

**Comment by isocolsky** [comment]
Worked flawlessly in Android livenet. Great job!
A couple of notes:
- I think the link "send to my wallets" needs a better UI design. It doesn't look like something that can be tapped.
- After a wallet is selected from the list, the flow continues as if I had scanned an address. Do you think it would be useful to show some kind of note stating that the address belongs to the wallet I selected?

File: public/views/walletHome.html [hunk 1]
```diff
@@ -327,6 +327,11 @@ <h4 class="title m0">
             <div ng-hide="home._paypro || home.hideAddress">
               <div class="row collapse">
 
+              <div class="right size-12 m0 text-gray" ng-click="openWalletsModal(index.otherWallets)" ng-if="index.otherWallets">
```
Comment (cmgustavo) [inline public/views/walletHome.html]: You could add `pointer` class to do more user friendly for desktop versions

**Comment by cmgustavo** [comment]
LGTM. manual test in iOS.

**Comment by matiu** [comment]
updated.

**Pull request merged**